README.md

```diff
@@ -1,2 +1,87 @@
-# BuyingCatalogueDocumentService
-A service for retrieving files from Azure blob storage
+# BuyingCatalogueDocumentService - Service architecture for the NHS Digital Buying Catalogue Document Service .Net Core application.
+
+## IMPORTANT NOTES!
+**You can use either the latest version of Visual Studio or .NET CLI for Windows, Mac and Linux**.
+
+### Architecture overview
+This application uses <b>.NET core</b> to provide an API capable of running on Linux or Windows.
```

Comment:
I think it might be better if we use 'pure' markdown rather than inline html, e.g. `**.NET core**` like in Important Notes.